### PR TITLE
travis, removes mentions of travis

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

**Review by gnott — APPROVED** [review]
Hopefully the line which was installing pandoc will not be missed, but we don't need that for testing on Jenkins currently because I think it uses the dockerised one.

**Comment by lsh-0** [comment]
There is no rush to merge these. If you think something is missing then hold off on approval and go investigate.

**Comment by gnott** [comment]
It's all good, that was just easier to install the test requirement on the travis-ci machine to use the Debian package, but in eLife tests and in prod we use a dockerised one.

**Pull request merged**
